storage/google/cloud/storage/client.py

```diff
@@ -194,7 +194,7 @@ def lookup_bucket(self, bucket_name):
         except NotFound:
             return None
 
-    def create_bucket(self, bucket_name):
+    def create_bucket(self, bucket_name, requester_pays=None):
```

Comment:
@tseaver I don't disagree, the question is:

- Should it also be in `Client.bucket()`?
- Should we just accept `**kwargs` in the `Bucket` constructor and then try to match them with properties